### ISSUE TITLE
how to add logging for trades sent

### ISSUE THREAD

**Issue by qsels** [PogoDigitalism/RobloxTradeAuthenticator#3]
I've been playing around with this yesterday and wondering how i add logging to sending a trade via this. I am talking about the ones in the screenshot returned by the screenshot below by the trade api. 

![image](https://github.com/PogoDigitalism/RobloxAuthenticator/assets/67203605/8ba35fb9-bf9f-473b-981d-739f42aefa3a)

(i am new to github so sorry if this is ugly/bad)

**Comment by PogoDigitalism** [comment]
Good idea, I should add better logging support. However, what do you mean for now? could you be a little more specific?

**Comment by qsels** [comment]
Like what code would i put to see what the json of the trade request returns, ex if i were to send a trade with requests and print the pesponse.json it would return either one of the values in the screenshot or the id of the trade if it went through. 

-I am asking how i log the json if the send trade request.

**Comment by PogoDigitalism** [comment]
Currently i did not add support for logging it through a function, so. I advice you edit the code yourselves to add support for that. You would have to:

use the logging module by Python, or just print the statements/add your own logic

you can add this functionality below the following lines:

In `authenticator.py`:
Use `line 70` to add your logic (this is for the synchronous authenticator class) and get the response from variable `resp`. 

This variable represents a `requests.Response` object which you can do whatever with. If you want to solely log the json of this (i dont recommend that) (non-async): log/print/whatever `resp.json()`

**Comment by PogoDigitalism** [comment]
Let me know if this solved your issue!
